keyboards/at101ish/at101ish.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "quantum.h" 
+
+#define _____ KC_NO
+
+/* Every possible switch positions on the PCB. Depending on plate and keycap choice, some of these positions will be blocked by other keys. */
+#define LAYOUT_all( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313, K0314,   K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412, K0413, K0414,                          K0415, K0416, K0417, K0418, \
+	K0501, K0502, K0503, K0504, K0505, K0506, K0507, K0508, K0509, K0510, K0511, K0512, K0513, K0514,          K0515,          K0516, K0517, K0518, K0519, \
+	K0601, K0602, K0603,                         K0604,                   K0605, K0606, K0607, K0608,   K0609, K0610, K0611,   K0612,        K0613         \
+) { \
+	{ K0101, K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110 }, \
+	{ K0111, K0112, K0113, K0114, K0115, K0116, _____, _____, K0321, K0221 }, \
+	{ K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210 }, \
+	{ K0211, K0212, K0213, K0214, K0215, K0216, K0217, K0218, K0219, K0220 }, \
+	{ K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310 }, \
+	{ K0311, K0312, K0313, K0314, K0315, K0316, K0317, K0318, K0319, K0320 }, \
+	{ K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410 }, \
+	{ K0411, K0412, K0413, K0414, _____, _____, K0415, K0416, K0417, K0418 }, \
+	{ K0501, K0502, K0503, K0504, K0505, K0506, K0507, K0508, K0509, K0510 }, \
+	{ K0511, K0512, K0513, K0514, K0515, _____, K0516, K0517, K0518, K0519 }, \
+	{ K0601, K0602, K0603, _____, _____, _____, _____, K0604, _____, _____ }, \
+	{ K0605, K0606, K0607, K0608, K0609, K0610, K0611, K0612, K0613, _____ }  \
+}
+
+#define LAYOUT_at101( \
+	K0101,        K0102, K0103, K0104, K0105, K0106, K0107, K0108, K0109, K0110, K0111, K0112, K0113,   K0114, K0115, K0116, \
+	K0201, K0202, K0203, K0204, K0205, K0206, K0207, K0208, K0209, K0210, K0211, K0212, K0213, K0214,   K0215, K0216, K0217,   K0218, K0219, K0220, K0221, \
+	K0301, K0302, K0303, K0304, K0305, K0306, K0307, K0308, K0309, K0310, K0311, K0312, K0313, K0314,   K0315, K0316, K0317,   K0318, K0319, K0320, K0321, \
+	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417, K0418, \
```

Comment:
Only `LAYOUT_all` would use `K0418`.

```suggestion
	K0401, K0402, K0403, K0404, K0405, K0406, K0407, K0408, K0409, K0410, K0411, K0412,    K0414,                              K0415, K0416, K0417,        \
```